### DISCUSSION TITLE
Enhancing Coaxial Rotor System Performance and Calculating High Mach Condition Effects

### DISCUSSION THREAD

**Discussion by Tyrant-sudo** [byuflowlab/FLOWUnsteady#97] [General]
I've set up a system with counter-rotating coaxial rotors, but I'm not pleased with its performance, particularly the false thrust generated by the downstream rotor. What steps can I take to improve this situation? Additionally, is it feasible to calculate rotor performance under high Mach conditions, and if so, how can I go about it?

**Comment by Burning1997** [comment]
I have the same problem.
